subsys/fs/shell.c

```diff
@@ -147,20 +265,24 @@ static int cmd_read(int argc, char *argv[])
 
 	err = fs_stat(path, &dirent);
 	if (err) {
-		printk("Failed to stat %s (%d)\n", path, err);
-		return err;
+		shell_fprintf(shell, SHELL_ERROR, "Failed to stat %s (%d)\r\n",
```

Comment:
error message: "Failed to obtain/get file/file-syetem object status"

Reply:
I understand that message was inherited :D

Reply:
yes, i didn't touch messages but i fixed that one on your request :)